po/pt.po

```diff
@@ -53,7 +53,7 @@ msgstr "Outros"
 
 #: ../src/Tweaks.vala:70
 msgid "Plank"
-msgstr "Plank"
+msgstr "Barra"
```

Comment:
In this case, "Plank" is actually the name of the dock :) I guess it could either be left as "plank", or does Barra imply correctly that it's the bottom dock?

Reply:
Yes, you're right, should leave it as Plank(Name of the program). 
("barra" means "bar"). Plank is nothing more than a mix of bar and a dock...
I've already changed that and made a few minor corrections.

Reply:
I think you might have only merged from master without pushing those new changes :)

Reply:
I only attached the translation (.zip file) at the beginning of this conversation
Sorry, my mistake....  I'm not very good at github.
I just do translations.
I think it's okay now